content/_index.html

```diff
@@ -5,89 +5,176 @@
-		</div>
-	</div>
-
-	<div class="lead">
-		<div class="image right">
-			<img src="icn-people.svg" alt="People Icon">
-		</div>
-		<div class="text">
-			<h4>Community</h4>
-			<p>We are an open and welcoming community of software developers, data scientists, and organizations that are working to make it easier to develop and deploy scalable ML workflows across the industry. <a href="/docs/about/contributing/">Start contributing.</a>
-		</div>
-	</div>
+  <div class="container">
+    <div class="card-deck">
+      <div class="card border-primary">
+        <img
+          src="logos/Jupyter.png"
+		  class="card-img-top"
+		  draggable="false"
+          style="padding: 2rem;"
+          alt="Jupyter Logo"
+        />
+        <div class="card-body bg-primary">
+          <h5 class="card-title text-light section-head">Notebooks</h5>
+          <p class="card-text text-light">
+            Kubeflow includes services to create and manage interactive <a target='_blank' class='text-light' href="https://jupyter.org/">Jupyter
+            notebooks</a>. You can choose custom images, and allocate specific
+            compute resources based on your data science needs. This
+            comprehensive experience allows you to create workflows locally, and
+            deploy them with ease.
+          </p>
+        </div>
+      </div>
+      <div class="card border-primary">
+        <img
+          src="logos/TensorFlow.png"
+		  class="card-img-top"
+		  draggable="false"
+          style="padding: 2rem;"
+          alt="TensorFlow"
+        />
+        <div class="card-body bg-primary">
+          <h5 class="card-title text-light section-head">
+            TensorFlow model training
+          </h5>
+          <p class="card-text text-light">
+            Kubeflow has a built-in <a target='_blank' class='text-light' href="https://www.tensorflow.org/">TensorFlow </a> job operator that you can use to
+            train a machine learning model with TensorFlow. The training
+            controller can be configured to use various compute resources (CPUs
+            or GPUs) and is compatible with various cluster sizes.
+          </p>
+        </div>
+      </div>
+      <div class="card border-primary">
+        <img
+          src="logos/Models.png"
+		  class="card-img-top"
+		  draggable="false"
+          style="padding: 2rem;"
+          alt="Model Serving image"
+        />
+        <div class="card-body bg-primary">
+          <h5 class="card-title text-light section-head">Model Serving</h5>
+          <p class="card-text text-light">
+            Kubeflow supports a <a target='_blank' class='text-light' href="https://www.tensorflow.org/">TensorFlow </a> Serving container to export trained
```

Comment:
Instead of linking to the specific framework's docs, we should link to the Kubeflow component page for each system. Readers can find their way to the framework from the Kubeflow component page.

So, in this first case, change this:

```
<a target='_blank' class='text-light' href="https://www.tensorflow.org/">TensorFlow </a> Serving
```

To this:

```
<a target='_blank' class='text-light' href="/docs/components/tfserving_new/">TensorFlow Serving</a>
```
(The name of the TF thing is "TensorFlow Serving", so I've moved the word "Serving" into the link alias.)

Please would you make similar changes to the other component links on this page.